src/pages/GenerateKeys/Option1.tsx

```diff
@@ -18,32 +19,50 @@ export const Option1 = ({
   validatorCount: number | string;
   os: string;
 }) => {
+  const cliLink = React.useMemo(() => {
+    if (os === 'linux')
+      return 'https://github.com/ethereum/eth2.0-deposit-cli/releases/download/v0.4.1/eth2deposit-cli-3f4a79a-darwin-amd64.tar.gz';
+    if (os === 'mac')
+      return 'https://github.com/ethereum/eth2.0-deposit-cli/releases/download/v0.4.1/eth2deposit-cli-3f4a79a-linux-amd64.tar.gz';
+    if (os === 'windows')
+      return 'https://github.com/ethereum/eth2.0-deposit-cli/releases/download/v0.4.1/eth2deposit-cli-3f4a79a-windows-amd64.zip';
+    return 'https://github.com/ethereum/eth2.0-deposit-cli/releases/download/v0.4.1/eth2deposit-cli-3f4a79a-windows-amd64.zip';
+  }, [os]);
+
+  const osName = React.useMemo(() => {
+    if (os === 'linux') return 'Linux';
+    if (os === 'mac') return 'MacOS';
+    if (os === 'windows') return 'Windows';
+    return 'default';
+  }, [os]);
+
   return (
     <div className="mt30">
       <Heading level={2} size="small" color="blueMedium" className="mb20">
         Option 1: Download command line app
       </Heading>
       <Text weight={500}>Download the deposit command line interface app</Text>
-      <Text>
-        Please find the zip/tarball file containing the latest deposit-cli
-        release on the{' '}
+      <Text className="mt5">
+        The easiest way to generate your deposit key files is by downloading the
+        CLI tool and following the instructions. You can also download the
+        deposit command line tool from the official Ethereum github account{' '}
         <Link
+          inline
           primary
           external
           to="https://github.com/ethereum/eth2.0-deposit-cli/releases/"
-          inline
         >
-          ethereum/eth2.0-deposit-cli
-        </Link>{' '}
-        GitHub page for your platform.{' '}
+          here
+        </Link>
+        .
       </Text>
-      <Alert variant="warning" className="my20">
-        <Text weight={500} color="yellowDarkest" className="my10">
-          Please make sure that you are downloading from the official Ethereum
-          Foundation GitHub account.
-        </Text>
-      </Alert>
-      <Text weight={500}>
+      <Link external to={cliLink} style={{ margin: '2rem auto' }}>
+        <Button
+          primary
+          label={<Text color="white">Download the CLI tool for {osName}</Text>}
+        />
+      </Link>
+      <Text weight={500} className="mt10">
```

Comment:
We have concerns if this pattern is "too convenient" for users so that they won't verify if the source is legit. If we want to add a direct link, IMHO then we *have* to include the checksum (#206) together.

Reply:
I agree @hwwhww. I don't want to merge this unless it's done in a verifiable way. We could skip this PR in this release until we feel comfortable that it's a safe option. I do think that this will reduce confusion if we can do this properly.

Reply:
@CarlBeek's checksum PR on the way: https://github.com/ethereum/eth2.0-deposit-cli/pull/133

Reply:
Checksum PR helps, not sure whether it solve the verification problem. People could just not verify the checksum. That said, most other apps ([eg geth](https://geth.ethereum.org/downloads/)) offer hosted binaries, so I guess it is ok

Reply:
EnvVars for checksums?

Reply:
I suggest also put download URLs to EnvVars: https://github.com/ethereum/eth2.0-deposit/pull/222
And also check the version.